src/alphatamp/approaches/abstract_explorers/random_explorer.py

```diff
@@ -0,0 +1,94 @@
+"""An random abstract plan explorer that returns a fixed length plan with random
+actions."""
+
+import random
```

Comment:
let's use `self._rng = np.random.default_rng(...)` instead of `random` to avoid global state, where multiple files are modifying the same random number generator